operator/operator.make

```diff
@@ -81,21 +61,27 @@ operator-generate-crd: controller-gen
 #   test configurations.
 operator-generate-bundle: _work/bin/operator-sdk-$(OPERATOR_SDK_VERSION) _work/kustomize operator-generate-crd
 	@echo "Generating operator bundle in $(BUNDLE_DIR) ..."
-	@_work/kustomize build --load-restrictor LoadRestrictionsNone $(MANIFESTS_DIR) | $< generate bundle --version=$(MAJOR_MINOR_PATCH_VERSION) \
-        --kustomize-dir=$(MANIFESTS_DIR) --output-dir=$(BUNDLE_DIR) --channels ${CHANNELS} --default-channel ${DEFAULT_CHANNEL}
+	rm -rf $(BUNDLE_DIR)
+	mkdir -p $(BUNDLE_DIR)
+	# Generate input YAML first. This might fail...
+	_work/kustomize build --load-restrictor LoadRestrictionsNone $(MANIFESTS_DIR) >$(BUNDLE_DIR)/manifests.yaml
```

Comment:
Directly piping the output caused me some head scratching because kustomize failed and I overlooked the one-line error message... Now make aborts directly.